RecoLocalCalo/CaloTowersCreator/src/CaloTowersCreator.cc

```diff
@@ -62,7 +63,8 @@ CaloTowersCreator::CaloTowersCreator(const edm::ParameterSet& conf) :
         conf.getParameter<double>("MomHBDepth"),
         conf.getParameter<double>("MomHEDepth"),
         conf.getParameter<double>("MomEBDepth"),
-        conf.getParameter<double>("MomEEDepth")
+        conf.getParameter<double>("MomEEDepth"),
+        conf.exists("HcalPhase") ? conf.getParameter<int>("HcalPhase") : 0
```

Comment:
how about FWCore/Framework/test/stubs/ToySTLProducers.cc
IntVectorProducer
and an example of multiple configurations in 
FWCore/Integration/test/testConsumesInfo_cfg.py